jobs/build/send-umb-messages/Jenkinsfile

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env groovy
+
+node {
+    properties([
+        disableConcurrentBuilds(),
+    ])
+
+    // we only care to publish messages for the following releases
+    releases = ["4-stable", "4.2.0-0.nightly"]
```

Comment:
@stevekuznetsov Can I ask why the ci stream is excluded? jw

Reply:
@stevekuznetsov ping - can you explain why ci stream is excluded? Thanks

Reply:
@johnbieren @p3ck sorry, just saw this -- the JIRA ticket asked for QE to be alerted for new OCP nightly candidates. We make a CI candidate dozens of times a day so that might be too noisy for the same level of messaging but if we need it, we can add it.